docker/dev.yml

```diff
@@ -39,6 +39,17 @@ services:
     <<: *minion-db-network
   backend:
     <<: [*backend-conf, *minion-db-network] 
+    extra_hosts:
```

Comment:
Drats; the indentation is incorrect here.  If tests pass, then I'll correct that next.

Reply:
There seem to be some ephemeral test failures related to downloading NPM packages; I have a fix prepared for the indentation, but will pause for now.

Reply:
This appears to have been [resolved](https://github.com/npm/cli/issues/8203), as most recently reported at 17:13:17 UTC (current time is: 18:53:29 UTC).

Reply:
> Drats; the indentation is incorrect here. If tests pass, then I'll correct that next.

No, I think it was correct originally after all.  The `<<` symbols in the YAML confused me; they're a more-or-less normal key name within the `backend` service object, other than the fact that `<<` is the syntax for a [merge key](https://yaml.org/type/merge.html).

I'll revert ea86a592de758bff51daebd9b7819e2ce15c4ae6.